config/default.go

```diff
@@ -43,17 +43,18 @@ var defaultConfig = &Config{
 	Registry: Registry{
 		Backend: "consul",
 		Consul: Consul{
-			Addr:          "localhost:8500",
-			Scheme:        "http",
-			KVPath:        "/fabio/config",
-			TagPrefix:     "urlprefix-",
-			Register:      true,
-			ServiceAddr:   ":9998",
-			ServiceName:   "fabio",
-			ServiceStatus: []string{"passing"},
-			CheckInterval: time.Second,
-			CheckTimeout:  3 * time.Second,
-			CheckScheme:   "http",
+			Addr:               "localhost:8500",
+			Scheme:             "http",
+			KVPath:             "/fabio/config",
+			TagPrefix:          "urlprefix-",
+			Register:           true,
+			ServiceAddr:        ":9998",
+			ServiceName:        "fabio",
+			ServiceStatus:      []string{"passing"},
+			CheckInterval:      time.Second,
+			CheckTimeout:       3 * time.Second,
+			CheckScheme:        "http",
+			CheckTLSSkipVerify: false,
```

Comment:
It is, but then how do I expose the parameter? 

https://github.com/fabiolb/fabio/pull/268/files#diff-f1ea9ed43b0b0ed74a6cf0f71dad3196R165

Do you want `defaultConfig.Registry.Consul.CheckTLSSkipVerify` just to be `false` in the link above?